SPC Categorial Outlook Display.py

```diff
@@ -38,80 +42,64 @@ def display_spc_outlook(outlook_data):
 
     # Overlay US state outlines
-    
-    ### ENTER STATE OUTLINE FILE DIRECTORY BELOW (Use / and not \) ###
-    states = gpd.read_file('') 
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    states_shapefile = os.path.join(current_directory, 's_11au16.shp')  
+    states = gpd.read_file(states_shapefile)  
     states.plot(ax=ax, facecolor='none', edgecolor='black', lw=0.5)
 
     # Read and plot the U.S. interstate highways shapefile
-
-    ### ENTER HIGHWAY SHAPEFILE BELOW (Use / and not \) ###
-    highways_shapefile = ''
+    highways_shapefile = os.path.join(current_directory, 'USA_Freeway_System.shp')
     highways_gdf = gpd.read_file(highways_shapefile)
-    highways_gdf.plot(ax=ax, color='red', linewidth=1)
+    highways_gdf.plot(ax=ax, color='red', linewidth=0.6)
 
     # Remove latitude and longitude axes labels and ticks
-
-    # DO NOT TOUCH
     ax.set_xticks([])
     ax.set_yticks([])
     ax.set_xticklabels([])
     ax.set_yticklabels([])
 
     # Set the x and y limits for CONUS
     # Base x & y: x (-125, -66) y (23, 50)
-
-    # Use the X and Y limits to create custom views
-    ax.set_xlim([-125, -66])
-    ax.set_ylim([24, 50])
+    ax.set_xlim([-125, -55])
+    ax.set_ylim([23, 50])
 
     # Remove the box around the plot
-    
-    # NO NOT TOUCH
     ax.spines['top'].set_visible(False)
     ax.spines['right'].set_visible(False)
     ax.spines['bottom'].set_visible(False)
     ax.spines['left'].set_visible(False)
 
     # Remove the title
-
-    # Code to change title, leave blank for no title
     plt.title('')
 
     # Add the header image
-    ### ENTER HEADER FILE LOCATION BELOW (Use / and not \) ###
-
-    header_img = plt.imread('')  
-    header_img = OffsetImage(header_img, zoom=0.25)
-    ab = AnnotationBbox(header_img, (0.265, 1.1), xycoords='axes fraction', frameon=False)
+    header_img = plt.imread(os.path.join(current_directory, 'WTUS_SPC_Banner_nobg.png'))  
+    header_img = OffsetImage(header_img, zoom=0.4)
+    ab = AnnotationBbox(header_img, (0.3, 1.1), xycoords='axes fraction', frameon=False)
     ax.add_artist(ab)
 
     # Save the plot as an image
-    ### ENTER FILE PATH BELOW (Use / and not \) ###
+    output_directory = create_output_directory()
+    output_filename = 'spc_cat_outlook.png'
+    output_path = os.path.join(output_directory, output_filename)
+    plt.savefig(output_path, dpi=500, bbox_inches='tight')
 
-    output_directory = ''
-    output_filename = 'spc_outlook.png'
-    output_path = output_directory + output_filename
-    plt.savefig(output_path, dpi=300, bbox_inches='tight')
-    
     # Show the plot
     plt.show()
 
 # Function to determine the color for each outlook category
-# Use this to determine colors
 def get_outlook_color(outlook_type):
     colors = {
         'TSTM': 'lightgreen',
         'MRGL': 'green',
         'SLGT': 'yellow',
         'ENH': 'orange',
         'MDT': 'red',
-        'HIGH': 'pink'
+        'HIGH': 'magenta'

```

Comment:
Updated the color of the HIGH risk to better visualise the color used on other platforms